docs/eslint.md

```diff
@@ -21,7 +27,11 @@ yarn add --dev prettier eslint-plugin-prettier
 }
 ```
 
-We also recommend that you use [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) to disable all the existing formatting rules. It's a one liner that can be added on-top of any existing ESLint configuration.
+## Turn off ESLint's formatting rules
+
+Whether you run Prettier via ESLint or run both tools separately, you probably only want to hear about each formatting issue once, and you especially don't want ESLint to complain about formatting "issues" which are simply a different preference than what Prettier does.
+
+So you'll probably want to disable all of ESLint's formatting rules (while keeping around non-formatting rules that Prettier doesn't care about).  The easiest way to do this is to use [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier). It's a one liner that can be added on-top of any existing ESLint configuration.
```

Comment:
Technically, you don't have to disable **all** formatting rules, there are some rules that might not conflict with prettier, see: https://github.com/prettier/eslint-config-prettier#special-rules

Reply:
Added another paragraph at the bottom.